### DISCUSSION TITLE
Can we build this provider for Windows?

### DISCUSSION THREAD

**Discussion by gowdamanil** [openssl-projects/pkcs11-provider#458] [Q&A] [answered]
Hello team,

We wanted to use pkcs11-provider for windows, Is it possible to build same source for windows?

Thanks,
Ani

**Comment by manison** [comment]
See https://github.com/latchset/pkcs11-provider/discussions/177#discussioncomment-7380986

**Reply by gowdamanil** [reply]
hello @manison 

**Thank you** for the code. I was able to build the code successfully. I  have one more doubt. This is related to pkcs module. Can we also build tpm2-pkcs11 module for windows as well?? Does this module also contain the solution file somewhere? Because I do not see it in git code here. https://github.com/tpm2-software/tpm2-pkcs11

Or can u guide me how to build it for windows.

**Reply by manison** [reply]
Sorry, don't have any knowledge of tpm2-pkcs11.

**Comment by simo5** [comment, marked as answer]
We are not currently handling windows compatibility, that is not to say we do not want to, but we simply do not have the expertise/bandwidth to do so at this time.

Should someone volunteer their expertise to enable windows builds and then help keep them working I am not opposed to have that support in pks11-provider.
